pages/edit/[id].js

```diff
@@ -0,0 +1,52 @@
+import { useRouter } from "next/router";
+import OutivityForm from "@/components/OutivityForm";
+import { useEffect, useState } from "react";
+
+export default function UpdateOutivityDetails({ outivities, onEditOutivity }) {
+  const router = useRouter();
+  const { id } = router.query;
+  const [selectedImage, setSelectedImage] = useState("");
+  const [outivity, setOutivity] = useState(null);
+
+  useEffect(() => {
+    const currentOutivity = outivities.find((outivity) => outivity.id === id);
+    if (currentOutivity) {
+      setOutivity(currentOutivity);
+      setSelectedImage(currentOutivity.image);
+    } else {
+    }
```

Comment:
I don't think we need the else here. Maybe you had something going on previously?

Reply:
Yes, you're right!
This is simply a leftover and no longer needed.